server/src/main/scala/file/FileClient.scala

```diff
@@ -0,0 +1,56 @@
+package fr.dopolytech.polydrive
+package file
+
+import com.typesafe.config.ConfigFactory
+import io.minio.http.Method
+import io.minio.{BucketExistsArgs, GetPresignedObjectUrlArgs, MinioClient}
+
+import java.util.concurrent.TimeUnit
+
+class FileClient {
+  val config = ConfigFactory.load()
```

Comment:
Use the config given from Server

```suggestion
class FileClient() {
  val config = ConfigFactory.load(config: Config)
```